e2e/sdk.erc721.e2e.test.ts

```diff
@@ -68,6 +68,18 @@ describe('SDK - contract interaction (deploy, load and mint)', () => {
     expect(response2.total).toBeGreaterThan(response.total);
     expect(response2.assets[0].metadata).toEqual(undefined);
 
+    await wait(
+      async () => {
+        const responseGetCollectionByWallet: any = await sdk.api.getCollectionsByWallet({
+          walletAddress: '0x3bE0Ec232d2D9B3912dE6f1ff941CB499db4eCe7',
+        });
+        return responseGetCollectionByWallet.collections !== null;
+      },
+      600000,
+      1000,
+      'Waiting for NFT metadata to be available',
+    );
+
```

Comment:
you waiting for the collection to be available, but you are not making the assertion for it after. so you should add the "expect" also to veryfy that the collection data is within the response

Reply:
we don't need to wait here